docs/learn-faq.md

```diff
@@ -81,6 +81,88 @@ This script helps you quickly identify which validator is the lowest-staked and
+be domain-specific blockchains. Ethereum is instead a smart contract chain that
+focuses on maximizing the decentralization of its architecture. For more on the
+differences between the two chains see the more comprehensive [comparison article](learn-comparison-ethereum).
+
+In order to be an Ethereum killer, Polkadot would have to be actively working
+against the progression of the Ethereum technology and ecosystem, which is not
+true. Instead, Web3 Foundation, the holders of the treasury from the DOT sales, 
+promotes and funds technology that benefits the web3 world as a whole.
+
+### On-chain governance is flawed, it can never work
+
+Blockchain governance is a very young field, and no one knows what works and
+what doesn't -- yet. Polkadot's governance arises from a couple key observations,
+1) blockchains need some way to adapt and evolve and 2) the only measure of value
+that is intrinsic to a blockchain is its native coin and therefore the best
+measure of stakeholders for the system are the token holders. There are of
+course caveats to the above two observations and that is why Polkadot's governance
+is still in its experimental phases. Polkadot takes the approach that it will try
+out best methods and observe how they work and tweak them until they're optimal.
+
+### Polkadot's governance is plutocratic
+
+Since Polkadot uses "coin-voting" that allows token holders to vote based on the
+amount of their holdings, some claim that Polkadot must be plutocratic. 
+
+However, Polkadot's governance is actually more fine-grained than this and has
+built in mechanisms that increase resiliency over plutocracy and cartels. One of
+these mechanisms is the council, an elected body that can make motions that will
+be voted on as refernda with a bias toward passing. The council acts as the voice
+for passive token holders or other users of the system that cannot vote. Another
+mechanism that can be employed by any token holder is called _conviction voting_
+and allows any toke holder to amplify their vote for stronger voting power. These
+mechanisms, and more, provide counter weights to the tendency for a simple token
+voting system to devolve into a plutocracy.
+
+For more on Polkadot's governance please see the [governance page](learn-governance).
+
+### Eth2.0 is better because it has 200,000+ validators and Polkadot has 1000
+
+The answer to this myth is still at this point totally unclear. The argument is
+that Eth2.0 will be more decentralized because it supports more validators than
+Polkadot. On Eth2.0, the barriers to entry for being a validator can be lower
+because the requirement is only 32 ETH, while on Polkadot it can be many multiples
+of this. On the flip side Polkadot has a more robust nomination system for electing
+the validator set that Eth2.0 lacks. It is still uncertain whether a decentralized
+network can support the network bandwidth and overhead of consensus messages being
```

Comment:
Not *that* unclear. 100k+ testnets have been running live in production for a while now. This is a weak argument imo. If we can't do this because we don't have BLS aggregation yet, then we should be honest about it. 4k validators is "an easy target" as per [sigp lighthouse blogs](https://lighthouse.sigmaprime.io/update-23.html), so saying a 1000 is our estimate makes it sound weird. To properly defend against this myth, we should:

- explain why we don't have more validators (BLS)
- explain that we don't have beacon nodes, and our validator nodes are full nodes. Ethereum will suffer from a beacon node deficit and incentivization problem, and this should be made clear.
- explain that with remote signing coming up, we will be able to deploy keys on micro-hardware and secure enclaves away from validator nodes, and with such a setup we approach the setup of Eth2 - validator nodes can then be likened to beacon nodes, and validators are in effect these remote keys. With that in mind, we could also theoretically run hundreds of thousands of such keys purely because running them is so light on resources. 
- The real bottleneck is the bandwidth and the logic that our validator nodes have to do - our validators are already gearing up to work with parachains, and the ops they do now will not differ much from the ops they'll be doing when parachains roll around. This will change with XCMP adding bandwidth weight, but parachains should work as is. Ethereum2 on the other hand is just testing the beacon chain, no shards whatsoever, and will initially only use the beacon chain to finalize eth1. This is a major difference in computing responsibilities of our software.